library/ssl_debug_helpers.h

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Since this is no longer generated, should it be moved to `include/mbedtls`, as is the case with `error.h` (included in `error.c`) and `version.h` (included in `version_features.c`)?

Reply:
No, these are for internal use only, so they're declared in `library`, like the other `library/*.h`.